yt_dlp/extractor/ilpost.py

```diff
@@ -23,46 +23,85 @@ class IlPostIE(InfoExtractor):
             'url': 'https://www.ilpost.it/wp-content/uploads/2023/12/28/1703781217-l-invasione-pt1-v6.mp3',
             'timestamp': 1703835014,
             'upload_date': '20231229',
+            'description': 'md5:54c5f12fb5b90f6c7cca6476a0802a99',
             'duration': 2495.0,
             'availability': 'public',
             'series_id': '235598',
-            'description': '',
+            'thumbnail': 'https://www.ilpost.it/wp-content/uploads/2023/12/22/1703238848-copertina500x500.jpg',
         },
     }]
 
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
 
-        endpoint_metadata = self._search_json(
-            r'var\s+ilpostpodcast\s*=', webpage, 'metadata', display_id)
-        episode_id = endpoint_metadata['post_id']
-        podcast_id = endpoint_metadata['podcast_id']
-        podcast_metadata = self._download_json(
-            endpoint_metadata['ajax_url'], display_id, data=urlencode_postdata({
-                'action': 'checkpodcast',
-                'cookie': endpoint_metadata['cookie'],
-                'post_id': episode_id,
-                'podcast_id': podcast_id,
-            }))
-
-        episode = traverse_obj(podcast_metadata, (
-            'data', 'postcastList', lambda _, v: str(v['id']) == episode_id, {dict}), get_all=False)
-        if not episode:
-            raise ExtractorError('Episode could not be extracted')
+        try:
+            episode = self._search_nextjs_data(webpage, display_id)['props']['pageProps']['data']['data']['episode']['data'][0]
+            episode_id = episode['id']
+        except KeyError:
+            raise ExtractorError('Failed to extract episode')
 
         return {
-            'id': episode_id,
+            'id': str(episode_id),
             'display_id': display_id,
-            'series_id': podcast_id,
-            'vcodec': 'none',
             **traverse_obj(episode, {
-                'title': ('title', {str}),
-                'description': ('description', {str}),
-                'url': ('podcast_raw_url', {url_or_none}),
+                'series_id': ('parent', 'id', {int_or_none}),
```

Comment:
`series_id` needs to be a string
```suggestion
                'series_id': ('parent', 'id', {int}, {str_or_none}),
```